config/webpack.config.dev.js

```diff
@@ -347,14 +344,16 @@ module.exports = {
     // You can remove this if you don't use Moment.js:
     new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
   ],
-  // Some libraries import Node modules but don't use them in the browser.
-  // Tell Webpack to provide empty mocks for them so importing them works.
-  node: {
-    dgram: 'empty',
-    fs: 'empty',
-    net: 'empty',
-    tls: 'empty',
-    child_process: 'empty',
+  // Set the target as `electron-renderer` so that packages provided by electron
+  // such as fs, path, electron are ignored by webpack
+  target: 'electron-renderer',
+  externals: {
+    'ps-tree': {
+      commonjs: 'ps-tree',
+    },
+    'electron-store': {
+      commonjs: 'electron-store',
+    },
```

Comment:
If we aren't bundling these modules, will they work in the production build? Guessing `electron-renderer` knows about these, to defer to Node?

Reply:
Ah, sorry for the delay, didn't see that you responded so quickly!

I cloned your fork and when I try running it locally, I get the following error:

<img width="1121" alt="screen shot 2018-08-13 at 8 49 16 am" src="https://user-images.githubusercontent.com/6692932/44032720-f9c779de-9ed5-11e8-97d8-2164ccbeb3ba.png">

My guess is that because Webpack isn't attempting to bundle these packages, they aren't smart enough to run the `require` at runtime? Is webpack smart enough to transpile these to `require` statements that Node can understand but _not_ try to resolve them during build?

Reply:
hum for some reasons, the webpack version in here doesn't like `packageName: { commonjs: packageName}` so I changed it for one that does work (I was on another branch with another version of webpack and just cherry picked the commit, sorry)